lambda/index.mjs

```diff
@@ -39,7 +39,7 @@ export default async function (plop) {
         {
           type: "addMany",
           destination: "./",
-          base: "templates/config",
+          base: "templates/repo-config",
```

Comment:
`repo-config` isn't a correct name for what these templates do.

In the frontend repo I referred to them as `npm` ... which is more accurate but still could do with some improvement.

Reply:
It's not just npm config (unlike in frontend) - there's eslint and jest config in there as well. `repo-config` is the best I could come up with. I'm open to suggestions but I don't think `npm` is a good name in this case

Reply:
https://github.com/govuk-one-login/ipv-cri-templates/tree/main/frontend/templates/npm - This is the similar folder in the `frontend` repo. 

Repo config is not accurate as a) there is already set of actions to init a repository  & b) not every repo will need node or npm.

Perhaps `node` is a better name, as that's a little bit wider than npm?

If you're renaming it here, you should also rename it in the frontend repo, so things are consistent.